clearpath_viz/rviz/robot.rviz

```diff
@@ -0,0 +1,385 @@
+          Position Transformer: XYZ
+          Queue Size: 10
+          Selectable: true
+          Size (Pixels): 3
+          Size (m): 0.03
+          Style: Flat Squares
+          Topic: /scan
+          Use Fixed Frame: true
+          Use rainbow: true
+          Value: true
+        - Alpha: 1
+          Autocompute Intensity Bounds: true
+          Autocompute Value Bounds:
+            Max Value: 10
+            Min Value: -10
+            Value: true
+          Axis: Z
+          Channel Name: intensity
+          Class: rviz/PointCloud2
+          Color: 255; 255; 255
+          Color Transformer: RGB8
+          Decay Time: 0
+          Enabled: true
+          Invert Rainbow: false
+          Max Color: 255; 255; 255
+          Max Intensity: 4096
+          Min Color: 0; 0; 0
+          Min Intensity: 0
+          Name: PointCloud2
+          Position Transformer: XYZ
+          Queue Size: 10
+          Selectable: true
+          Size (Pixels): 3
+          Size (m): 0.01
+          Style: Flat Squares
+          Topic: /camera/depth/points
+          Use Fixed Frame: true
+          Use rainbow: true
+          Value: true
+        - Acceleration properties:
+            Acc. vector alpha: 0.5
+            Acc. vector color: 85; 0; 255
+            Acc. vector scale: 0.05
+            Derotate acceleration: false
+            Enable acceleration: true
+          Axes properties:
+            Axes scale: 1
+            Enable axes: true
+          Box properties:
+            Box alpha: 1
+            Box color: 85; 0; 255
+            Enable box: false
+            x_scale: 0.4
+            y_scale: 0.4
+            z_scale: 0.4
+          Class: rviz_imu_plugin/Imu
+          Enabled: false
+          Name: Imu
+          Topic: /imu/data
+          Value: false
+      Enabled: true
+      Name: Sensing
+    - Class: rviz/Group
+      Displays:
+        - Class: rviz/Group
+          Displays:
+            - Class: rviz/Marker
+              Enabled: true
+              Marker Topic: /exploration_polygon_marker
+              Name: Exploration Marker
+              Namespaces:
+                {}
+              Queue Size: 100
+              Value: true
+            - Alpha: 0.7
+              Class: rviz/Map
+              Color Scheme: costmap
+              Draw Behind: true
+              Enabled: true
+              Name: Exploration Costmap
+              Topic: /explore_server/explore_costmap/costmap
+              Value: true
+          Enabled: true
+          Name: Exploration
+        - Alpha: 0.7
+          Class: rviz/Map
+          Color Scheme: map
+          Draw Behind: true
+          Enabled: true
+          Name: Static Map
+          Topic: /map
+          Value: true
+        - Alpha: 0.7
+          Class: rviz/Map
+          Color Scheme: costmap
+          Draw Behind: true
+          Enabled: true
+          Name: Global Costmap
+          Topic: /move_base/global_costmap/costmap
```

Comment:
These topics look like ROS1 topics.

Reply:
They are ROS 1 topics.  This will get updated for the new topics.  I don't think robot should have any mapping away.